lib/ical/event.js

```diff
@@ -49,6 +49,14 @@ ICAL.Event = (function() {
     if (options && options.exceptions) {
       options.exceptions.forEach(this.relateException, this);
     }
+
+    if (this.component.parent && !this.isRecurrenceException()) {
```

Comment:
I think we should expand this condition to make sure `options.exceptions` is not set. In this case the caller has specific intent w.r.t. the exceptions and may not want the parent exceptions to be automatically added or has already done so for legacy reasons.

Also, can you add some text to the header of the function that explains that exceptions will automatically be set in relation to the parent of the component, if the component is not a recurrence exception?